filters/generic_filters.txt

```diff
@@ -2321,4 +2321,4 @@ vt.edu#@#.scrollto
 what.thedailywtf.com#@#.post-tools
 wikihow.com#@##actionbar
 wsj.com#@##share-target
-zdnet.com#@#.share-bar-wrapper
\ No newline at end of file
+zdnet.com#@#.share-bar-wrapper
```

Comment:
No new line at end of file? Bad practice.

Reply:
@gwarser thanks for pull, but also had question about why bad practice? The alphabetizer sort that I use always removes the blank line at end of file. I would need to see about overriding that behaviour if it really would cause issues removing the new line at end of file.

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline